src/winforms/toga_winforms/widgets/canvas.py

```diff
@@ -290,7 +290,7 @@ def write_text(self, text, x, y, font, draw_context, *args, **kwargs):
             _, height = self.measure_text(line, font)
             origin = PointF(x, y + full_height - height)
             draw_context.current_path.AddString(
-                line, font_family, font_style, float(height), origin, StringFormat()
+                line, font_family, font_style.value__, float(height), origin, StringFormat()
```

Comment:
I get the following error when I try to run the canvas example from this branch:
``` 
line, font_family, font_style.value__, float(height), origin, StringFormat()
AttributeError : 'int' object has no attribute 'value__'
``` 
I haven't run toga in a long while. Do I need to update something?

Reply:
`font_style` is returning 0 for me, so the sample app works when I remove `.value__` from this line:
`line, font_family, font_style, float(height), origin, StringFormat()`

Reply:
This is the only part that gives me pause, too. I haven't had a chance to boot up my Windows box, but `value__` looks like an attribute name that isn't meant to be touched. 

@saroad2 Can I ask how you found this one? Is it a documented feature of Pythonnet? Or something you found by poking around with pdb?

Reply:
Oh, sorry. I assumed that since I used editable install (with `pip install -e`), all the changes would be applied and dependencies would be installed.
This change does indeed fix the Canvas example because font_style now is now returning `System.Drawing.FontStyle` instead of the numerical value.
I guess this is caused by this breaking change from the[ 'Unreleased Changelog'](https://github.com/pythonnet/pythonnet/blob/master/CHANGELOG.md#changed):

> BREAKING: disabled implicit conversion from C# enums to Python int and back. One must now either use enum members (e.g. MyEnum.Option), or use enum constructor (e.g. MyEnum(42) or MyEnum(42, True) when MyEnum does not have a member with value 42).

P.S. It might require many more changes to Enum handling

Reply:
Ah - a change in implicit enum conversion make sense. Is `.value__` the offical interface for retrieving the value of an Enum (either in C#, or in Python.net)?

Reply:
[This PR ](https://github.com/pythonnet/pythonnet/pull/1661)implements an `__int__` Enum instance method, so we should be able to use `int(font_style)`. But it was merged 21 days ago, and we probably don't have it in our version of pythonnet?

Reply:
Hey @freakboy3742  and @obulat, let me explain (I'll add this explanation to the PR too, for future readers):

About two months ago we started using an alpha version of *Pythonnet 3.0*. This version now treats enums as objects instead of integers. the only way to get the integer value of the enum is using the `value__` attribute.

In the meanwhile, I opened an [issue](pythonnet/pythonnet#1585) for fixing this problem, and suggested adding the `__int__` method for getting `value__`. While *Pythonnet* did implement this ability in pythonnet/pythonnet#1661, it is not available in the alpha version.

In order for *Toga* to keep working until the next version of *Pythonnet* is released, I added this PR. once they publish their production version, I'll create a new PR and use the `__int__` method instead.

Reply:
Makes sense - thanks for the clarification (and for chasing down the issue with Pythonnet)!